src/bluesearch/entrypoint/database/topic_extract.py

```diff
@@ -168,6 +171,30 @@ def run(
                     },
                 }
             )
+
+    elif source in {"biorxiv", "medrxiv"}:
+        for path in inputs:
+            logger.info(f"Processing {path}")
+            topic, journal = extract_info_from_zipfile(path)
+            all_results.append(
+                {
+                    "source": journal,
```

Comment:
I think the `source` identified for the other cases was to simply put the source itself (i.e. `biorxiv`, `medrxiv`, `pmc`, `pubmed`, `arxiv`).

Reply:
That is a possibility. However, what is tricky is that no matter if the user provided biorxiv` or `medrix` the same code is executed (= the extraction logic is identical for both of them). And that is why I just wanted to make sure that we actually find the journal inside of the `xml` rather then just copying what the CLI `source` was. 

I hope that makes sense. Maybe this logic is stupid. What do you think?

Reply:
Also see the implementation in #539 I guess it would make sense to be consistent on this one.

Reply:
@Stannislav makes sense. I am about to review it actually. However, maybe we can do it in a separate PR once both this PR and the other one are merged.

Reply:
Yeah, I also don't mind merging this PR first and then adjusting mine